### DISCUSSION TITLE
Need to Change 'PVC' Size

### DISCUSSION THREAD

**Discussion by xCodeOwl** [OT-CONTAINER-KIT/redis-operator#114] [Q&A] [answered]
No matter what I try (edit the Helm chart '_values.yaml_' file, set a parameter on the '_Helm upgrade_' command line) the original size of '_1Gi_' specified in the '_values.yaml_' file of the _**redis-cluster**_ Helm chart does not change.
Interestingly enough, running the '_Helm upgrade_' command with the '_--dry-run_' option produces output that shows the desired PVC size.
However, running '_kubectl -n redis-name-space get pvc_' shows two PVC's in pending state. In the (yaml/json formatted) manifest output from either PVC the claim-size continues to be shown to be set to '**1Gi**'.
Please advise what I should be doing to change the PVC claim-size.

By the way, the '_redis-operator_' pod is in the _**running**_ state, while the '_redis-cluster-leader-o_' and '_redis-cluster-follower-o_' pods along with their associated PVC's are in the _**pending**_ state. Also, I have an NFS-Provisioner deployed and running in my cluster which supports the _StorageClass_ "_**nfs-client**_". This _StorageClass_ is marked to be the **_default_**.

**Comment by iamabhishek-dubey** [comment, marked as answer]
We would suggest to do the changes directly at PVC level by editing them or patching them.
